lib/WP_Auth0_LoginManager.php

```diff
@@ -111,11 +127,10 @@ public function login_auto() {
 			// Do not redirect log out action.
 			|| ( isset( $_GET['action'] ) && 'logout' === $_GET['action'] )
 			// Do not redirect Auth0 login processing.
+			// TODO: This should be removed as this page is no longer used for callbacks.
 			|| null !== $this->query_vars( 'auth0' )
-			// Do not redirect if already authenticated.
-			|| is_user_logged_in()
 		) {
-			return;
+			return false;
```

Comment:
This is used in a hook call so return value is never used ... expect in tests now!